maps/src/main/java/org/odk/collect/maps/layers/OfflineMapLayersPickerViewModel.kt

```diff
@@ -0,0 +1,46 @@
+package org.odk.collect.maps.layers
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import org.odk.collect.androidshared.async.TrackableWorker
+import org.odk.collect.async.Scheduler
+import org.odk.collect.settings.SettingsProvider
+import org.odk.collect.settings.keys.ProjectKeys
+
+class OfflineMapLayersPickerViewModel(
```

Comment:
Actually, I'm happy that we no longer keep the code that is not shared in the shared ViewModel. Not using a shared ViewModel for communication between fragments is also the direction we were thinking about in https://github.com/getodk/collect/pull/6243#discussion_r1662129936